### ISSUE TITLE
Support Svelte and Vue

### ISSUE THREAD

**Issue by robertheessels** [meouwu-dev/css-var-color-decorator#1]
https://github.com/meouwu-dev/css-var-color-decorator/blob/d712049731a7b3200bb66e80c0b12ec8a788ba32/src/listener/index.ts#L6

Great extension. Thanx!

Would be awesome if you could support <style> blocks in HTML, Svelte and Vue files!

**Comment by robertheessels** [comment]
And maybe also Javascript files, like tailwind.config.js.

**Comment by robertheessels** [comment]
And maybe also Stylus files (*.styl). :-)

**Comment by meouwu-dev** [comment]
Hi, sorry for the late reply. 

I have published a new version (v1.2.0). You can add support for other languages by adding them to the setting `cssVarColorDecorator.supportedLanguages`.

For example, if you want to add support for html, you can add it like this:

```json
"cssVarColorDecorator.supportedLanguages": ["css", "scss", "tailwindcss", "html"]
```

**Comment by robertheessels** [comment]
@meouwu-dev Works great. Thank you so much!!!
